src/components/shared/footer/footer.jsx

```diff
@@ -1,12 +1,14 @@
+import clsx from 'clsx';
 import Link from 'next/link';
+import PropTypes from 'prop-types';
 import React from 'react';
 
 import Button from 'components/shared/button';
 import MENUS from 'constants/menus';
 import Logo from 'images/logo.inline.svg';
 
-const Footer = () => (
-  <footer className="safe-paddings">
+const Footer = ({ isBordered }) => (
```

Comment:
```suggestion
const Footer = ({ withBorder }) => (
```
I think we could simplify the name of the parameter for better understanding - `isBordered` => `withBorder`
Wdyt?

Reply:
@andrewgolovanov I`m agree!)